src/components/List_Products.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import { getItems, saveItem } from '../lib/api';
+
+function List_Products() {
+  const [Items, setItems] = useState(null);
+  const [ItemName, setItemName] = useState(null);
+
+  /* Get Items */
+  useEffect(() => {
+    getItemsData();
+  }, []);
+  const getItemsData = async () => {
+    const Itm = await getItems();
```

Comment:
Typo: `Itm`. I guess it should be `Item` 😄 .

Reply:
Names changed in both spelling and casing. :)